srcpkgs/pinephone-uboot/template

```diff
@@ -0,0 +1,105 @@
+# Template file for 'pinephone-uboot'
+pkgname=pinephone-uboot
+version=0.0.20200904
+revision=1
+archs="aarch64*"
+create_wrksrc=yes
+hostmakedepends="flex cross-or1k-none-elf dtc python3 python3-devel bc swig"
+depends="u-boot-tools"
+conf_files="/etc/default/${pkgname}-config"
+short_desc="U-Boot for Pinephone with Crust support"
+maintainer="John Sullivan <jsullivan@csumb.edu>"
+license="GPL-2.0-only, BSD-3-Clause"
+homepage="http://www.denx.de/wiki/U-Boot/WebHome"
+
+_tag_uboot=crust-2020-09-04
+_commit_armtf=2bdb4611ad6efdb3da2ff0d2de7fee4c43e95aeb
+_commit_crust=e63106c70bc4ddfa3575e4f1f3667de0795252a8
+
+_uboot=u-boot-$_tag_uboot
+_armtf=arm-trusted-firmware-$_commit_armtf
+_crust=crust-$_commit_crust
```

Comment:
We usually try to use released tags rather than direct commits. Seems like it should be easy to do for crust at least.

Reply:
It seems the latest stable version of crust is only in master (check [postmarketOS's](https://gitlab.com/postmarketOS/pmaports/-/blob/master/main/crust/APKBUILD) build script) so I can't change to a tag for the moment. However, I've asked crust's developer if a new tag can be added. I'll come back here if I hear anything.

Reply:
most recent crust tag seems recent enough from my perspective. ATF is fine as is.

Reply:
Cool. I'll just leave this here for posterity: https://github.com/crust-firmware/crust/commit/65fa3f23621aa3a0f4ea48465df629201de954ba This is tag 0.2, with the latest commit being from last year. I'm fairly certain there's been quite a bit done since then.

Reply:
Does this mean we expect a new push soon? If yes, we should wait a bit, meanwhile, I have some comment.

Reply:
@sgn I tried pinging smaeul (crust dev) on irc but they didn't come around. I can try again.

Reply:
Let's wait for a few more days :-p I was about to fix the `-j $makejobs` myself if I didn't see this :-p
Hope you don't mind.

Reply:
@sgn Already fixed the makejobs thing, I'll try leaving an issue on their repo too

Reply:
Sorry I assumed that the 0.2 tag, which was made on July 12, included all of the commits up to July 11. My bad here.

Reply:
No trouble, it should probably be tagged anyway. I was just inclined to go with the flow of what other maintainers were doing. sgn is probably right to hold off to see if anything changes

Reply:
@sgn @CameronNemo got a response from [smaeul:](https://view.matrix.org/room/!ToPtLQdvveNEbXcHqE:fam-ribbers.com/$RXeBCeUBvyco7tcUQPlFWtc7qLig2Yq8Xn3syRNycMg?via=matrix.org&via=feneas.org&via=kde.org) `
jjsullivan: crust master is what you should be using, yes. I don't plan to tag 0.3 until I have fixed the known issues with the current version (crashing on a couple of devices)`

So the new tag is probably a bit far off from now, but master is working with the pinephone. The 'other devices' are other Allwinner products using the same embedded controller.